tsconfig.production.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
Should we/can we instruct `ncc` to use this file? Do we need to, since we point it to a file (an entrypoint)?

Reply:
As far as I could see `ncc` looks at the `tsconfig.json` file and there is no option to override this

Reply:
I added an issue to tackle this here: https://github.com/ridedott/auto-merge-action/issues/18